sites/svelte.dev/src/routes/(authed)/repl/[id].json/+server.js

```diff
@@ -31,41 +35,47 @@ function munge(files) {
 }
 
 export async function GET({ params }) {
-	if (!examples) {
-		const res = await fetch(`${PUBLIC_API_BASE}/docs/svelte/examples`);
+	// Currently, these pages(that are in examples/) are prerendered. To avoid making any FS requests,
+	// We prerender examples pages during build time. That means, when something like `/repl/hello-world.json`
+	// is accessed, this function won't be run at all, as it will be served from the filesystem
+
+	try {
+		const examples_data = get_examples_data();
 		examples = new Set(
-			(await res.json())
+			get_examples_list(examples_data)
 				.map((category) => category.examples)
 				.flat()
 				.map((example) => example.slug)
 		);
-	}
 
-	if (examples.has(params.id)) {
-		const res = await fetch(`${PUBLIC_API_BASE}/docs/svelte/examples/${params.id}`);
+		if (examples.has(params.id)) {
+			const example = get_example(examples_data, params.id);
 
-		if (!res.ok) {
-			return new Response(await res.json(), {
-				status: res.status,
-				headers: { 'Content-Type': 'application/json' },
+			return json({
+				id: params.id,
+				name: example.title,
+				owner: null,
+				relaxed: false, // TODO is this right? EDIT: It was example.relaxed before, which no example return to my knowledge. By @PuruVJ
```

Comment:
i think _all_ examples are relaxed (meaning the iframe gets `allow-same-origin`)

Reply:
Gotcha!